<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="windowStateLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="awakeBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Awake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="copyrightLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="currentStatusLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Misc</string>
    </property>
    <addaction name="actionSource_Codes"/>
    <addaction name="separator"/>
    <addaction name="action_Copyright_2022_proscawards_All_rights_reserved_Made_with_and"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionInfo">
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="action_Copyright_2022_proscawards_All_rights_reserved_Made_with_and">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Â©Copyright 2022 proscawards.</string>
   </property>
  </action>
  <action name="actionSource_Codes">
   <property name="text">
    <string>Source Codes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
